src/Workleap.Extensions.Http.Authentication.ClientCredentialsGrant/TracingHelper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+
+namespace Workleap.Extensions.Http.Authentication.ClientCredentialsGrant;
+
+internal static class TracingHelper
+{
+    private const string ActivityName = "ClientCredentials";
+
+    private const string ClientIdTagName = "client_credentials.client_id";
```

Comment:
I checked for OTel naming conventions and the built-in tags use snake case (ex: `trace.trace_id`, `user_agent.original`).

Microsoft's Kiota usesthe reverse DNS notation (`com.microsoft.kiota.chaos_handler_triggered`) but I haven't found similar examples.